2996_reflection/reflection.md

```diff
@@ -3067,6 +3078,29 @@ Since namespace aliases are now entities, but their declarations are not definit
 
 ### [basic.def.odr]{.sref} One-definition rule {-}
 
+Add `$splice-expression$`s to the set of potential results of an expression in paragarph 3.
+
+::: std
+[3]{.pnum} An expression or conversion is _potentially evaluated_ unless it is an unevaluated operand ([expr.context]{.sref}), a subexpression thereof, or a conversion in an initialization or conversion sequence in such a context. The set of _potential results_ of an expression `$E$` is defined as follows:
+
+- [#.#]{.pnum} If `$E$` is an `$id-expression$` ([expr.prim.id]{.sref}) [or a `$splice-expression$` ([expr.prim.splice])]{.addu}, the set contains only `$E$`.
+- [#.#]{.pnum} [...]
+
+[This set is a (possibly-empty) set of `$id-expression$`s [and `$splice-expression$`s]{.addu}, each of which is either `$E$` or a subexpression of `$E$`.]{.note}
+
+:::
+
+Break bullet 4.1 into sub-bullets, modify it to cover splicing of functions, and replace [basic.lookup] with [over.pre] since the canonical definition of "overload set" is relocated there by this proposal:
+
+::: std
+- [4.1]{.pnum} A function is named by an expression or conversion if it is the selected member of an overload set ([[basic.pre]]{.rm} [[over.pre]]{.addu}, [over.match], [over.over]) in an overload resolution performed as part of forming that expression or conversion, unless it is a pure virtual function and [either]{.rm} the expression[\ ]{.addu}
+
+  - [#.#.#]{.pnum} is not an `$id-expression$` naming the function with an explicitly qualified name[,]{.addu}
+  - [[#.#.#]{.pnum} is not a `$splice-expression$`,]{.addu} or
+  - [#.#.#]{.pnum} [the expression]{.rm} forms a pointer to member ([expr.unary.op]).
```

Comment:
I think move "the expression" into each bullet. Even better, "A function is named by an expression or conversion [`$E$`]{.addu} if [...]" and then:

> * [#.#.#]{.pnum} [`$E$`]{.addu} is not an `$id-expression$` [...],
> * [#.#.#]{.pnum} [`$E$` is not a `$splice-expression$`, or]{.addu}
> * [#.#.#]{.pnum} [the expression]{.rm} [`$E$`]{.addu} forms a pointer to member.

?

Reply:
Done.